OxfordBodleian/Juel-Jensen/BDLaethd18.xml

```diff
@@ -12,7 +12,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title xml:lang="gez" xml:id="title1">ሄኖክ፡</title>
                 <title xml:lang="gez" corresp="#title1" type="normalized">Henok</title>
-                <title xml:lang="en" corresp="#title1">Book of Enoch</title>
+                <title xml:lang="en" corresp="#title1">Book of <persName ref="PRS3814Enoch">Enoch</persName></title>
```

Comment:
I wonder if we need this markup. Do we want this instance of Enoch to be listed in the index? Is not the relevant entity here the "Book of Enoch", not the person Enoch?

Reply:
This is a very important point for us to decide and it will need to be done consistently throughout to avoid a lot of copy-editing further down the line. I don't have any strong feelings, I think the decision should rest with Massimo, @sologebre and @thea-m as: a) you are responsible for that section of work; b) it is your line of expertise.

I will just say that marking up all instances will be a LOT of additional work for encoders (as you will have to include PERS ID for every author/subject mentioned in the title of a text). And that it will also be more work removing all those repeat page references (e.g. _Mark, 2, 62 ,62 ,62 ,80_) which Thea told us cannot be encoded out of the index.

Reply:
Okay, if we can take advantage of the possibilities of markup to automate the separate indexing of persons and works and avoid redundant references in the indices, then it is better to remove these markups.
Is it fine with you?

Reply:
Yes, I think that would be a good solution